src/hooks/api/index.ts

```diff
@@ -63,17 +63,10 @@ export function useAccountsMonthlyTotals(): SWRResponse<queries.MonthlyTotals> {
     fetcher(PriceDB.getTodayQuotes, '/api/prices/today'),
   );
 
-  // This makes sure that every time new accounts appear we recalculate
-  // the monthly totals.
-  //
-  // Not only this but re-using the same key has super poor performance
-  // for some reason
```

Comment:
so apparently the bad performance was due to apexcharts re-rendering multiple times on each key mutation. Now with chartjs is way better so we can use single key which is more manageable. When we make this query more granular (re-compute totals per account) we may re-add this.